src/main/resources/swagger.api/mosaic.yaml

```diff
@@ -27,27 +23,76 @@ paths:
                 type: string
               example: "10001-1"
         '400':
-          description:
-            Bad request, e.g. malformed request body or query parameter.
-            Details of the error (e.g. name of the parameter or line/character number
-            with malformed data) provided in the response.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/errors"
+          $ref: "#/components/responses/BadRequest"
         '500':
-          description: Internal server error, e.g. due to misconfiguration
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/errors"
+          $ref: "#/components/responses/InternalServerError"
+
+  ### Configuration API ###
+  /configuration:
+    description: Configuration API
+    get:
+      summary: Get Mosaic configuration
+      operationId: getConfiguration
+      responses:
+        '200':
+          $ref: "#/components/responses/MosaicConfigurationResponse"
+        '404':
+          $ref: "#/components/responses/NotFound"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+    post:
+      summary: Save Mosaic configuration
+      operationId: saveConfiguration
+      requestBody:
+        $ref: "#/components/requestBodies/MosaicConfigurationBody"
+      responses:
+        '201':
+          $ref: "#/components/responses/MosaicConfigurationResponse"
+        '400':
+          $ref: "#/components/responses/BadRequest"
+        '409':
+          $ref: "#/components/responses/Conflict"
+        '422':
+          $ref: "#/components/responses/UnprocessableEntity"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+    put:
+      summary: Update Mosaic configuration
+      operationId: updateConfiguration
+      requestBody:
+        $ref: "#/components/requestBodies/MosaicConfigurationBody"
+      responses:
+        '204':
+          $ref: "#/components/responses/NoContent"
+        '400':
+          $ref: "#/components/responses/BadRequest"
+        '404':
+          $ref: "#/components/responses/NotFound"
+        '422':
+          $ref: "#/components/responses/UnprocessableEntity"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+    delete:
+      summary: Delete Mosaic configuration
+      operationId: deleteConfiguration
+      responses:
+        '204':
+          $ref: "#/components/responses/NoContent"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+
 components:
   schemas:
     MosaicOrder:
       type: object
       properties:
         name:
           $ref: "MosaicOrder"
+    MosaicConfiguration:
+      type: object
+      properties:
+        name:
+          $ref: "MosaicConfiguration"
```

Comment:
There reference is causing api lint error, can we specify its schema and then reference like `$ref: "schemas/userTenant.yaml#/UserTenant"`